cattle/compute/__init__.py

```diff
@@ -63,6 +47,27 @@ def instance_deactivate(self, req=None, instanceHostMap=None,
                                                         progress)
         )
 
+    def get_instance_host_from_map(self, instanceHostMap=None):
+        instance = instanceHostMap.instance
+        host = instanceHostMap.host
+
+        try:
+            host.clusterConnection = \
+                instanceHostMap.data.fields['clusterConnection']
+            log.debug('clusterConnection = %s', host.clusterConnection)
+            host.caCrt = \
+                instanceHostMap.data.fields['caCrt']
+            host.clientCrt = \
+                instanceHostMap.data.fields['clientCrt']
+            host.clientKey = \
+                instanceHostMap.data.fields['clientKey']
+            log.debug('caCrt=%s clientCrt=%s clientKey=%s',
+                      host.caCrt, host.clientCrt, host.clientKey)
+        except (KeyError, AttributeError):
+            pass
```

Comment:
@sonchang Son, what happens when keyError/attributeError are caught, and the caller stack retrieves instance/host w/o cert information set?